components/QuestChain/QuestChainV0Page.tsx

```diff
@@ -701,8 +701,8 @@ export const QuestChainV0Page: React.FC<QuestChainV0PageProps> = ({
                         review
                       </Flex>
                       <NextLink
-                        as={`/chain/${questChain.chainId}/${questChain.address}/review`}
-                        href={`/chain/[chainId]/[address]/review`}
+                        as={`/${questChain.chainId}/${questChain.address}/review`}
+                        href={`/[chainId]/[address]/review`}
```

Comment:
```suggestion
                        href={`/[network]/[address]/review`}
```

The `[chainId]` page no longer exists right? For both v0 and v1 its the same url, isn't it?

Reply:
true